kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -587,37 +669,97 @@ define([
                 );
         },
 
+        renderImportButton: function() {
+
+            let importButton = $('<button></button>')
+            .addClass('kb-staging-table-import__button btn btn-xs btn-primary')
+            .text('Import Selected');
+
+            this.$elem.find('div.kb-staging-table-import').append(importButton);
+
+            /* 
+                By default import button is disabled until the user selects a data type 
+            */
+            this.disableImportButton();
+        },
+
+        disableImportButton: function() {
+
+            this.$elem.find('button.kb-staging-table-import__button')
+            .addClass('kb-staging-table-import__button__disabled')
+            .tooltip({
+                title: 'Select a file/s to continue.',
+                delay: {
+                    show: Config.get('tooltip').showDelay,
+                    hide: Config.get('tooltip').hideDelay
+                }
+            })
+            .off('click');
+        },
+
+        enableImportButton: function() {
+            let stagingAreaViewer = this; 
+
+            this.$elem.find('button.kb-staging-table-import__button')
+            .removeClass('kb-staging-table-import__button__disabled')
+            .tooltip('disable')
+            .unbind('click')
```

Comment:
I'm pretty sure `unbind` is deprecated for jquery elements in favor of `off`.

Reply:
ah - ok, must be my old jquery brain here. i can update to off